src/miner_lora.erl

```diff
@@ -127,11 +137,15 @@ handle_info({tx_timeout, Token}, #state{packet_timers=Timers}=State) ->
     end,
     {noreply, State#state{packet_timers=maps:remove(Token, Timers)}};
 handle_info({udp, Socket, IP, Port, Packet}, #state{socket=Socket}=State) ->
+    maybe_mirror(State#state.mirror_socket, Packet),
```

Comment:
Care about the response here from gen_udp:send ?

Reply:
nope, fire and forget